packages/Chem/src/rendering/rdkit-cell-renderer.ts

```diff
@@ -6,13 +6,26 @@ import * as DG from 'datagrok-api/dg';
 import {_rdKitModule, drawErrorCross, drawRdKitMoleculeToOffscreenCanvas} from '../utils/chem-common-rdkit';
 import {IMolContext, getMolSafe} from '../utils/mol-creation_rdkit';
 import {RDModule} from '@datagrok-libraries/chem-meta/src/rdkit-api';
+import { ALIGN_BY_SCAFFOLD_TAG, FILTER_SCAFFOLD_TAG, HIGHLIGHT_BY_SCAFFOLD_TAG } from '../constants';
+
+export interface ISubstruct {
+  atoms?: number[],
+  bonds?: number[],
+  highlightAtomColors?: {[key: number]: number[]},
+  highlightBondColors?: {[key: number]: number[]}
+}
 
 interface IMolInfo {
   //mol: RDMol | null; // null when molString is invalid?
   molCtx: IMolContext;
-  substruct: any;
+  substruct: ISubstruct;
```

Comment:
Why does molInfo contain all of that? Perhaps a class should be renamed?

Reply:
Yes, I renamed it to IMolRenderingInfo (it contains information required to render molecule with highlighted scaffols)